pkg/pipelines/pipelines.go

```diff
@@ -342,6 +342,8 @@ func ToPipelineActivity(pr *v1beta1.PipelineRun, pa *v1.PipelineActivity, overwr
 	}
 
 	activities.UpdateStatus(pa, false, nil)
+
+	pa.Spec.Message = v1.ActivityMessageType(pr.Status.Conditions[0].Message)
```

Comment:
This was just to demonstrate that this is how u can get messages from tekton pipeline runs. This is brittle, and will break if you have more than one condition, or dont have a condition for example.